lib/evaluation/handlers/limitDisclosureEvaluationHandler.ts

```diff
@@ -72,18 +95,18 @@ export class LimitDisclosureEvaluationHandler extends AbstractEvaluationHandler
     };
   }
 
-  private createSuccessResult(idIdx: number, path: string) {
+  private createSuccessResult(idIdx: number, path: string, limitDisclosure: Optionality) {
     return this.getResults().push({
       input_descriptor_path: `$.input_descriptors[${idIdx}]`,
       verifiable_credential_path: `${path}`,
       evaluator: this.getName(),
-      status: Status.INFO,
+      status: limitDisclosure === Optionality.Required ? Status.INFO : Status.WARN,
```

Comment:
What about the optional case and a suite that supports selective disclosure?

Reply:
If the suite does not support selective disclosure it wouldn't reach that point. The error result will be generated at line 36.

Reply:
Well that is not what I was saying. I was saying what if the suite supports it and the optionality was set to Optional instead of required. We would start emitting warnings instead of infos

Reply:
My understanding is that if limit_disclosure is set to preferred, we should limit the disclosure of data and warn the user that it is optional.

Reply:
If it is optional and we see that the signature suite supports it, we have to (try) to use it IMO. This is all about protecting the user to leak as little privy info as possible

Reply:
But are we doing the selective disclosure when it is optional and not hit this path? Because if we do hit this path it means we now have a warning in the logs, which is confusing